Framework/PythonInterface/mantid/plots/__init__.py

```diff
@@ -39,6 +39,7 @@
 from mantid.plots.helperfunctions import get_normalize_by_bin_width
 from mantid.plots.scales import PowerScale, SquareScale
 from mantid.plots.utility import artists_hidden, MantidAxType, legend_set_draggable
+from mantidqt.widgets.plotconfigdialog.colorselector import convert_color_to_hex
```

Comment:
This is something I meant to mention more widely when I spotted it the other day but the framework code can't depend on any of the Qt modules. We have builds, like the conda build, that doesn't have mantidqt/workbench and these modules still need to function. It's entirely possible to plot with matplotlib without requiring Qt.

I made a commit to move a few things in my PR (#27698) as I thought it might be merged relatively quickly but perhaps we could apply that commit here and I'll rebase once this is through?

Reply:
This is the commit: https://github.com/mantidproject/mantid/pull/27698/commits/e9d5f5bc2397d4ce4dff118d5d7b2e95920d5293. You can apply it with
```
git fetch https://github.com/martyngigg/mantid.git  e9d5f5bc2397d4ce4dff118d5d7b2e95920d5293
git cherry-pick e9d5f5bc2397d4ce4dff118d5d7b2e95920d5293